addons/project/static/src/views/project_task_calendar/project_task_calendar_controller.js

```diff
@@ -3,6 +3,7 @@
 import { CalendarController } from "@web/views/calendar/calendar_controller";
 import { ProjectTaskCalendarFilterPanel } from "./project_task_calendar_filter_panel/project_task_calendar_filter_panel";
 import { DeleteSubtasksConfirmationDialog } from "@project/components/delete_subtasks_confirmation_dialog/delete_subtasks_confirmation_dialog";
+import { _t } from "@web/core/l10n/translation";
```

Comment:
Hello @bastvdn ,

Can you rebase your branches a last time and remove that line because it will be imported by another commit? 🙂 

Sorry about that. 🙂